### DISCUSSION TITLE
Model Serving API

### DISCUSSION THREAD

**Discussion by jtisbell4** [databricks/databricks-sdk-py#166] [Ideas]
Hi! Are there any plans to implement the [model serving API](https://docs.databricks.com/api/workspace/servingendpoints/create) in this client? The [current documentation](https://docs.databricks.com/machine-learning/model-serving/create-manage-serving-endpoints.html) only lists using the REST API and the UI as possible options for creating a new model serving endpoint. Thanks and happy to contribute!

**Comment by nfx** [comment]
@jtisbell4 it's already there - https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/service/serving.py#L479 - should be accessible as `w.serving_endpoints.` already

@PaulCornellDB can you make sure the documentation is updated?
